tests/components/decora_wifi/test_common.py

```diff
@@ -0,0 +1,189 @@
+"""Test the decora_wifi Common components."""
+
+from unittest.mock import patch
+
+from homeassistant.components.decora_wifi.common import (
+    DecoraWifiCommFailed,
+    DecoraWifiLoginFailed,
+    DecoraWifiPlatform,
+)
+from homeassistant.components.decora_wifi.const import LIGHT_DOMAIN
+
+from tests.components.decora_wifi.common import (
+    FakeDecoraWiFiResidence,
+    FakeDecoraWiFiResidentialAccount,
+    FakeDecoraWiFiSession,
+)
+
+USERNAME = "username@home-assisant.com"
+PASSWORD = "test-password"
+INCORRECT_PASSWORD = "incoreect-password"
+
+
+def test_DecoraWifiPlatform_init():
```

Comment:
All tests must be bare `async` tests.

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations